log/grpc.go

```diff
@@ -56,10 +57,14 @@ func UnaryServerInterceptor(logCtx context.Context, opts ...GRPCLogOption) grpc.
 		if o.disableCallLogging {
 			return handler(ctx, req)
 		}
+		print := Print
```

Comment:
Would it make sense to do this before the return so it's only done once at init time?

Reply:
Since the other options can be accidentally dynamic, I thought I'd follow suit with this one. Realistically there is no way for callers to influence our options struct however, so they're static even if we evaluate their fields on every call. Will hoist up to parent block.